src/components/CytoscapeGraph/CytoscapeToolbar.tsx

```diff
@@ -64,16 +64,16 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps>
     return (
       <div className={cytoscapeToolbarStyle}>
         <ButtonGroup>
-          <Button onClick={this.zoomIn}>
+          <Button onClick={this.zoomIn} title="zoom in">
```

Comment:
I think we should capitalize the first word, e.g. `Zoom in` and the same for the rest of the titles.

Reply:
ok, I mean OK